MANIFEST.in

```diff
@@ -4,19 +4,32 @@ include SYMLINK_CACHE.json
 include requirements.txt
 include shippable.yml
 include bin/ansible-test
```

Comment:
This should no longer be needed, as we should be including all files in `bin` already, correct?

Reply:
Yep, reoved bin/ansible-test i nthe latest version